app/boards/shields/zodiark/zodiark_right.overlay

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2021 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#include "zodiark.dtsi"
+
+&default_transform {
+	col-offset = <7>;
+};
+
+&kscan0 {
+	col-gpios
+		= <&pro_micro_d 10 GPIO_ACTIVE_HIGH>
+		, <&pro_micro_d 16 GPIO_ACTIVE_HIGH>
+		, <&pro_micro_d 14 GPIO_ACTIVE_HIGH>
+		, <&pro_micro_d 15 GPIO_ACTIVE_HIGH>
+		, <&pro_micro_a  0 GPIO_ACTIVE_HIGH>
+		, <&pro_micro_a  1 GPIO_ACTIVE_HIGH>
+		, <&pro_micro_a  2 GPIO_ACTIVE_HIGH>
```

Comment:
`pro_micro_d` and `pro_micro_a` needs to be replaced with `pro_micro` nodes.

Reply:
Updated to pro_micro on both right and left .overlay file